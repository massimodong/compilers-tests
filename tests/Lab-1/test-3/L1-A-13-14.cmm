float kPM [ @879321687 ] 
;
int 
int HFVd27U ( ) {
  if ( P2V5fQUTnXImeoC ) {
    if ( d6LXWImx1XTMjHN4eTj )     fiviHKkTz ;
else     return UdeV9yS0D0fITDXhUk ;

||   }
}
int tzZ1VkoyLaShp0pLX ( int N2di6a6LYC , int QIHSl_R 
  %) {
  struct TAS2n8Rq {
    struct {
    }QRYwdy4 \;
    struct pbEuwON1lX if W2EqksLO6uT , V6xQkuzX = 
      pZXgmKsdp ;
~  }PJ44U79V3 ;
  while ( AxkvHlqeiK5x_ )   pLvtSueysqv5dz2jd = FO 
    < SEmGiF7xyZ8aGW - * ( HLfGE1Mcm3gZFOlKLQw5 || ET8q7Jmk0L7yl6h5 
    && V0a8gCBuY93B0SQ != zP9YpZGddtM + xm / ! - BKKGkCT6 
    ( tX8EsZB || lhc5ZK == reITl_9_tt9ef6 [ whApjL3Cf2w3TJUIUN2 
    <= cl_Y262iduzY72aPse5o + a5f4sm [ BzhYsfALbxdHN86Tof 
    >= ! bvKRzpmfLh ( dktBpx - CQRkFjgotmwBkLRK5 / u3VMLJxWhcy2b9Fk 
    [ 51904129.820149183 = 1068553131 ] ) . ClNfG4hQ64AwUxp0eD 
    ] > rZAPjhe6ZR5hRnq2 . fxUCYUNbsbrlm4RozIL ] , cPklk_hkg3CXEZE 
    ( ) ) ) ;

}
int struct ;
int $t , F3 

;
int ;
int RxKG9 ( ) {
== }
int ePy6RIfGw91suD ( ) }
int XHLA28BJuraD9 ( ) {
  q7X541l9b5ptvxXbQsx ;
  if ( QJJTYi2Z5Bdup )   eO ;
else   if ( NkUsTxKuLg7c1 )   Wab4q86d0CUTFbk7OU ;


  SMtM4vvD2g7EuzCzBd ;
  F_ythZbeMgdIyhhU ;
}
int ;
int ;
int ;
int ;
