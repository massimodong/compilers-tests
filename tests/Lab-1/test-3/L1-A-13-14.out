Error Type A at Line 1
Error Type B at Line 4
Error Type B at Line 9
Error Type A at Line 12
Error Type A at Line 15
Error Type B at Line 16
Error Type A at Line 18
Error Type B at Line 20
Error Type B at Line 30
Error Type A at Line 31
Error Type B at Line 36
Error Type B at Line 37
